structuralcodes/geometry/_steel_sections.py

```diff
@@ -0,0 +1,1849 @@
+"""Utility functions for importing standard steel profiles."""
+
+import numpy as np
+from shapely import (
+    LineString,
+    Point,
+    Polygon,
+    get_geometry,
+    polygonize,
+    set_precision,
+)
+from shapely.affinity import scale, translate
+from shapely.geometry.polygon import orient
+from shapely.ops import linemerge, unary_union
+
+
+class IPE:
```

Comment:
Just thinking, wouldn't it be a quite simple fix to do a Marin integration of the circumference of a section and obtain the section moduli? It would be great to see these as a set of properties 👍

Reply:
Ok, done an attempt in 7a9e5c16ca08ef181a46fe6c905af22891167297

Reply:
Good! Just a thought, instead of computing the properties in the constructor, could we do a check in the getter for the property, and compute the properties only if the internal attribute is `None`?

Reply:
You are absolutely right! I started doing like that because they started being very simple computations and I did not foresee a potential problem of performances.
Doing like you are proposing is for sure more efficient: you can find the consequent refactor in commit 6302c3949a7bb2e868b29da0b38c1ee560cb07df.